demos/projects/PC/linux/port/azure_iot_flash_platform.c

```diff
@@ -0,0 +1,26 @@
+
+#include "azure_iot_flash_platform.h"
+
```

Comment:
Same here for the _OpenFile and _CloseFile

Reply:
I'm okay with waiting till we have a need for those.